### PR TITLE
fix double setState issue introduced by React 0.13

### PR DESCRIPTION
**Pull request by meta-meta**
this also looked to work with `cmp._reactInternalInstance._pendingStateQueue.pop()` but I thought it best not to disturb.

### CODE REVIEW

File: src/Cursor.js [hunk 1]
```diff
@@ -37,8 +37,10 @@ function update(cmp, path, operation, nextValue) {
   var nextState;
 
   if (path.length > 0) {
+    var q = cmp._reactInternalInstance._pendingStateQueue;
```
Comment (danielmiladinov) [inline src/Cursor.js]: I like this, it's clever. I also agree with you that it's better to peek at the last item rather than pop it off the queue.

Comment (dustingetz) [inline src/Cursor.js]: @meta-meta I'll merge this. But this looks not very future proof. Did you get any feel for what other alternatives there are? Maybe there is a better approach that react-cursor can take that might not need to inspect react internals now?

Comment (meta-meta) [inline src/Cursor.js]: @dustingetz I share this concern. I don't know that there is an api to access pendingState. I think this is a lateral change as we were illegally grabbing cmp._pendingState before.

Comment (dustingetz) [inline src/Cursor.js]: RIght, let me rephrase: does React .13 include improvements to the React API that make it no longer necessary for cursors to access the pending state?

Comment (meta-meta) [inline src/Cursor.js]: You can pass a function to setState now. I'm not sure how that helps this scenario but something to think about.
